packages/trackerless-network/src/logic/ContentDeliveryLayerNode.ts

```diff
@@ -65,7 +65,9 @@ export interface StrictContentDeliveryLayerNodeOptions {
     rpcRequestTimeout?: number
 }
 
-const RANDOM_NODE_VIEW_SIZE = 20
+export const DEFAULT_NODE_VIEW_SIZE = 20
```

Comment:
https://github.com/streamr-dev/network/blob/main/developer-docs/index.md#naming-counts

```suggestion
export const DEFAULT_NODE_VIEW_CONTACT_COUNT = 20
```

Could be part of a separate PR (in that case, add a bullet point to the "Future improvements" section)